src/profiling/d3d.rs

```diff
@@ -362,10 +261,10 @@ pub fn create_d3d_gpu_context(name: &str, renderer: &D3DSkiaRenderer) -> Box<dyn
         flags: GpuContextFlags::GpuContextCalibration as u8,
         type_: GpuContextType::Direct3D12 as u8,
     };
-    let namestring = CString::new(name).unwrap();
+    // let namestring = CString::new(name).unwrap();
     let name_data = ___tracy_gpu_context_name_data {
         context: ctx_id,
-        name: namestring.as_ptr(),
+        name: name.as_ptr() as *const i8,
```

Comment:
This is probably your crash with the profiler, the name needs to be null terminated, that's why a CString was used.

Reply:
Unfortunately, it still didn't work when I change back to `CString`. Even though, I'll reserve the back-change.

Reply:
Ok, let's deal with that later.